auth0/resource_auth0_organization.go

```diff
@@ -74,83 +69,32 @@ func newOrganization() *schema.Resource {
 						"assign_membership_on_login": {
 							Type:     schema.TypeBool,
 							Optional: true,
+							Default:  false,
 						},
 					},
 				},
-				Set: hash.StringKey("connection_id"),
 			},
 		},
 	}
 }
 
 func createOrganization(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
-	organization := buildOrganization(d)
+	organization := expandOrganization(d)
+
 	api := m.(*management.Management)
 	if err := api.Organization.Create(organization); err != nil {
 		return diag.FromErr(err)
 	}
 
 	d.SetId(organization.GetID())
 
-	d.Partial(true)
```

Comment:
Part of what caused the bug with the diffs I believe. The Partial func is a legacy func used when we had the SetPartial func as well (deprecated). This is not needed at all as on every command we run a refresh (read) even if we stop mid way here on the follow up plan or apply we'll read again and correct the state.